hack/deploy-photon-local.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+#
+# .-'_.---._'-.
+# ||####|(__)||   Protect your secrets, protect your business.
+#   \\()|##//       Secure your sensitive data with Aegis.
+#    \\ |#//                    <aegis.ist>
+#     .\_/.
+#
+
+cd ./k8s || exit
```

Comment:
These deploy scripts can be consolidated into a single one too; that also can be a follow-up pull request.